packages/rpc-graphql/src/__tests__/transaction-tests.ts

```diff
@@ -294,27 +294,23 @@ describe('transaction', () => {
                             data {
                                 message {
                                     instructions {
+                                            programId
                                         ... on ExtendLookupTableInstruction {
-                                            data {
-                                                lookupTableAccount {
-                                                    address
-                                                }
-                                                lookupTableAuthority {
-                                                    address
-                                                }
-                                                newAddresses
-                                                payerAccount {
-                                                    address
-                                                }
-                                                systemProgram {
-                                                    address
-                                                }
+                                            lookupTableAccount {
+                                                address
                                             }
-                                            meta {
-                                                program
-                                                type
+                                            lookupTableAuthority {
+                                                address
                                             }
-                                            programId
+                                            newAddresses
+                                            payerAccount {
+                                                address
+                                            }
+                                            systemProgram {
+                                                address
+                                            }
+                                            programName
+                                            instructionType
```

Comment:
Same as the last PR; is this not covered by `instruction.__type`?

Reply:
Perhaps I don't understand `__type` quite right.

Reply:
Made changes to this schema similar to this topic on accounts!